packedForest/src/forestTypes/basicForests/rfClassification/rfTree.h

```diff
@@ -217,12 +229,25 @@ inline bool isRightNode(){
 
 
 				inline int predictObservation(std::vector<T>& observation){
-					int currNode = 0;
+					int currNode = 1;
```

Comment:
if we skip over the root node, and jump straight to the second node of the tree don't we have problems?  or maybe i'm misunderstanding things